lib/reqTemplate.js

```diff
@@ -0,0 +1,109 @@
+"use strict";
+
+var URI = require('swagger-router').URI;
+var rbUtil = require('./rbUtil');
+var TAssembly = require('tassembly');
+
+/**
+ * Creates a function that sets a value at the path
+ * or deletes if undefined values is provided.
+ */
+function _setAtPathFun(path) {
```

Comment:
Minor nit: lose the `Fun` part in the name. It's clear this is a function, and `Fun` kind of makes me think of fun, not functions :)

Oh, and also, please document the `path` param in the description above.